Community Members/nylestroke.toml

```diff
@@ -0,0 +1,50 @@
+# Name you want to appear with. This can be whatever name/alias makes you most comfortable.
+# Mandatory
+name = "Vladyslav Potapenko"
+
+# File name of your profile picture if you uploaded one
+# If you provide a file, be mindful of its size
+# If you also add your github profile, you can use "GitHub" to use your GitHub profile picture
+# This can be any image that follows our code of conduct.
+# Optional
+profile-picture = "GitHub"
+
+# Short presentation of yourself. This has a limit of 180 characters.
+# Optional
+bio = "Full-Stack Software Engineer (Angular, NestJS). A Software-Developer Enthusiast."
+
+# Link to your sponsoring page, GitHub Sponsor, Patreon, Buy Me a Coffee, ...
+# Optional
+sponsor = ""
+
+# Social links
+
+# Will link to https://github.com/<username>
+# Optional
+github = "nylestroke"
+
+discord = "nylestroke#9372"
+# This allows direct linking to your profile for people with a shared server
+# https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-
+# Optional
+discord-userid = "668556214996566016"
+
+## Will link to https://<instance.com>/@<username>
+# Optional
+mastodon = "@nylestroke@instance.com"
+
+## Will link to https://twitter.com/<username>
+# Optional
+twitter = "nylestroke"
+
+## Will link to https://<username>.itch.io/
+# Optional
+itch-io = "nylestroke"
+
+## Will link to http://store.steampowered.com/developer/<developer_name>
+# Optional
+steam-developer = "nylestroke"
+
+## Free link, will be displayed as is
+# Optional
+website = "https://github.com/nylestroke"
```

Comment:
This link should already be present from the GitHub entry, do you want to have it twice?